contribution/translation/prevalidator.php

```diff
@@ -159,10 +179,15 @@ public function check_package($package, $reference_filepath)
 							}
 						}
 					}
+
+					// In the last step we have removed the license and index files if there were any. We'll just put a new one instead
+					$this->add_license_files($uploaded_lang_root . 'LICENSE', $reference_filepath);
```

Comment:
I'm not familiar with the way the license is checked. What happens if a translation with a wrong license is validated? Is it silently replaced with our copy of the GPLv2 here?

Reply:
Perhaps @Crizz0 can confirm, but in Poland I recall he said it must always be a copy of the GPL 2 license.